ssclj/jar/src/main/clojure/com/sixsq/slipstream/ssclj/app/main.clj

```diff
@@ -1,15 +1,27 @@
 (ns com.sixsq.slipstream.ssclj.app.main
   (:gen-class)
   (:require
-    [com.sixsq.slipstream.ssclj.app.server :refer [start register-shutdown-hook]]
-    [clojure.tools.logging :as log]))
+    [com.sixsq.slipstream.ssclj.app.server :refer [start stop]]))
+
+(def ^:const default-port 8200)
 
 (defn valid-port?
   "If the port number is valid, then returns the port itself;
-   otherwise returns nil."
+   otherwise returns false."
   [port]
-  (if (< 0 port 65536)
-    port))
+  (and (< 0 port 65536) port))
```

Comment:
Good point.  I've just integrated this into the `parse-port` function and added tests to make sure it has reasonable behavior for all inputs.  This change was pushed directly to master (859d3a5dddc1751f0966134be753d3242a60573c).